isis/scripts/downloadIsisData

```diff
@@ -11,9 +11,59 @@ import tempfile
 from shutil import which
 from os import path
 from collections import OrderedDict
+from pathlib import Path
+import sys
+import re
+
+exclude_list = [
+        '/a_older_versions/*',
+        '*/former_versions/',
+        'corrupt_files/*',
+        'zzarchive/**',
+        '*/original/',
+        'ck/prime_mission/*/extended_mission/*',
```

Comment:
I think this is the problem, there are two filters mashed into one. Also the asterisks seem to be messing up the filters. I guess rclone doesn't want asterisks. 


this seemed to be working for me. 

```python 
exclude_list = [
exclude_list = [
        '/a_older_versions/',
        '*/former_versions/',
        'corrupt_files/',
        'zzarchive/',
        '*/original/',
        'ck/prime_mission/',
        'extended_mission/',
        '*/prime_mission/',
        'ck/GEM/',
        'ck/save/',
        'spk/SAVE_SCS_2017-11-22/',
        'fk/release',
        'fk/Archive/'
    ]
```